src/contexts/user.context.js

```diff
@@ -0,0 +1,38 @@
+import React, { createContext, useContext, useState } from 'react'
+import PropTypes from 'prop-types'
+import eu from '../assets/images/eu.jpeg'
+
+const UserContext = createContext()
+
+export function UserProvider({ children }) {
+  const [name, setName] = useState('Rodrigo Eloy')
+  const [eMail, setEmail] = useState('rodrigo.cavalcanti@ccc.ufcg.edu.br')
+  const [course, setCourse] = useState('Ciência da computação')
+  const [registrationNumber, setRegistrationNumber] = useState('115478521')
```

Comment:
Acredito que poderia ser usado o nome `userID`. Só uma sugestão, não sei realmente é melhor para o contexto de vocês.

```suggestion
  const [userID, setUserID] = useState('115478521')
```